kong/db/migrations/operations/200_to_210.lua

```diff
@@ -29,13 +34,15 @@ local function cassandra_get_default_ws(coordinator)
   end
 
   if not rows
-     or not rows[1]
-     or not rows[1].id
+  or not rows[1]
+  or not rows[1].id
   then
```

Comment:
I think the original format is what we usually use, but since this condition are so short, writing them on a single line is even better.

```suggestion
  if not rows or not rows[1] or not rows[1].id then
```